gulpfile.js

```diff
@@ -201,22 +213,46 @@ gulp.task('vendor-scripts', () => beginPipe('./static/vendor/**/*.js')
 	.pipe(uglify())
 	.pipe(gulp.dest(`./build/${themeName()}/vendor`)));
 
+
+// =========================
+// COPY (Symlink) Files
+// =========================
+
 // copy other vendor files
-gulp.task('vendor-assets', () => beginPipe([
+gulp.task('vendor-assets', () => beginPipeLink([
 	'./static/vendor/**/*.*',
 	'!./static/vendor/**/*.js',
 	'!./static/vendor/**/*.{css,sass,scss}',
 ]).pipe(gulp.dest(`./build/${themeName()}/vendor`)));
 
 // copy vendor-optimized files
-gulp.task('vendor-optimized-assets', () => beginPipe(['./static/vendor-optimized/**/*.*'])
+gulp.task('vendor-optimized-assets', () => beginPipeLink(['./static/vendor-optimized/**/*.*'])
 	.pipe(gulp.dest(`./build/${themeName()}/vendor-optimized`)));
 
 // copy node modules
-const nodeModules = ['mathjax', 'font-awesome'];
-gulp.task('node-modules', () => Promise.all(nodeModules
-	.map(module => beginPipe([`./node_modules/${module}/**/*.*`])
-		.pipe(gulp.dest(`./build/${themeName()}/vendor-optimized/${module}`)))));
+const nodeModules = [
+	['mathjax', 'mathjax/MathJax.js'],
```

Comment:
can u create a single vendor js and css file for all vendor-optimized snippets instead of symlinking?

Reply:
you mean like we did with sentry?

Reply:
yop, using that we would no more have all the mathjax files to be copied here file by file